### ISSUE TITLE
Unexpected traversal order on home screen of new gallery

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#17375]
I’d expect the items to be traversierend from left to right, from top to bottom. Instead, after the first item of the second row, the first item of the third row is focused (should be second item of first row). 

This was observed on iOS, iPhone X. 

/cc @yjbanov @HansMuller

**Comment by HansMuller** [comment]
On Android the traversal order is as-expected.
